src/mono-runtimes/mono-runtimes.targets

```diff
@@ -128,6 +129,16 @@
         Include="@(_MonoUtility->'$(_MSBuildDir)\%(Filename).pdb')"
     />
   </ItemGroup>
+  <ItemGroup>
+    <_MonoBuildUtility Include="mono-api-html.exe" />
```

Comment:
These are more "interesting", in that the mono bundle we produce is extracted under e.g. `bin/Debug`, and doesn't contain files for e.g. `bin/BuildDebug`.

Thus, my earlier idea of copying these files into `bin/BuildDebug` is in fact a bad idea.

Doh.

We have two options here:

 1. Treat them the same as `@(_MonoUtility)`, and place them into e.g. `bin/Debug/lib/xamarin.android/xbuild/Xamarin/Android`.

 2. Add a "new top-level" e.g. `bin/Debug/util` and copy them there.

(1) is certainly easier, and placing files there *won't* mean they're included in the installer (which would require updating `build-tools/installers/create-installers.targets`).

I'm not sure if there's an "obvious" name for (2).